From 3a01a06804ace8de4ea01ffde93ded6b7b1ab8de Mon Sep 17 00:00:00 2001
From: Chris Wilson <chris@chris-wilson.co.uk>
Date: Tue, 1 Feb 2011 14:15:55 +0000
Patch-mainline: 2.6.39
References: fate#310916
Git-commit: b6f7833b9712c0c01f94cc3a518dc62b07bd610b
Subject: [PATCH 2230/2588] drm/i915: Include 'i915_error_state' hint for when
 the GPU catches fire

Signed-off-by: Chris Wilson <chris@chris-wilson.co.uk>
(cherry picked from commit b6f7833b9712c0c01f94cc3a518dc62b07bd610b)

Signed-off-by: Takashi Iwai <tiwai@suse.de>
---
 drivers/gpu/drm/i915/i915_irq.c |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/drivers/gpu/drm/i915/i915_irq.c b/drivers/gpu/drm/i915/i915_irq.c
index e612fb9..1c5c135 100644
--- a/drivers/gpu/drm/i915/i915_irq.c
+++ b/drivers/gpu/drm/i915/i915_irq.c
@@ -787,7 +787,8 @@ static void i915_capture_error_state(struct drm_device *dev)
 		return;
 	}
 
-	DRM_DEBUG_DRIVER("generating error event\n");
+	DRM_INFO("capturing error event; look for more information in /debug/dri/%d/i915_error_state\n",
+		 dev->primary->index);
 
 	error->seqno = dev_priv->ring[RCS].get_seqno(&dev_priv->ring[RCS]);
 	error->eir = I915_READ(EIR);
-- 
1.7.6

